### PR TITLE
Allow event tracks to go over date line

### LINKED ISSUE #1568: Events that cross the dateline cross the entire map, from east to west
**Describe the bug**
Tropical Cyclone Mona currently crosses the dateline on 2019-01-06. Instead of wrapping around, the event track crosses the entire map, from east to west.

**To Reproduce**
1. Go to 'Events'
2. Click on 'Tropical Cyclone Mona 2019-01-06'
3. See the event track from 2019-01-05 to 2019-01-06 cross the entire map from east to west.

**Expected behavior**
Would be nice if the event track wrapped around the other way. 

**Screenshots**
screen shot 2019-01-07 at 5 03 02 pm
screen shot 2019-01-07 at 5 08 48 pm

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser Chrome

### PR DESCRIPTION
**Pull request by Benjaki2**
## Description

Fixes #1568.

Events that cross the dateline cross the entire map, from east to west

- [x] Make event track go over date line
- [x] Add new testable event over date line to `mock/events_data.json-20170530`

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- [x] I have added necessary documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules (if applicable)

## Further comments

If this is a relatively large or complex change, start a discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

@nasa-gibs/worldview

### CODE REVIEW

File: web/js/natural-events/track.js [hunk 1]
```diff
@@ -203,6 +207,33 @@ export default function naturalEventsTrack(models, ui, config) {
...
+/**
+ * Determine if track point is across the date line from the
+ * selected point
+ * @param {Array} activeCoord Coordinate array of selected track point
+ * @param {Array} nextCoord Coordinate to test against active coord
+ * @return {Boolean}
+ */
+const crossesDateLine = function(activeCoord, nextCoord) {
+  return (
+    Math.abs(Math.abs(activeCoord[0]) - Math.abs(nextCoord[0])) <
```
Comment (Benjaki2) [inline web/js/natural-events/track.js]: @edplato  @mike-mcgann is there a better way to do this?

Comment (mike-mcgann) [reply web/js/natural-events/track.js]: One way would to assume that no two points would be separated by more than 180 degrees. Assume cross if abs(x1 - x2) > 180. 

I don't think what you have works when x1 = -1 and x2 = 1:

- abs(abs(-1) - abs(1)) < abs(-1 - 1) 
- abs(1 - 1) < abs(-2) 
- 0 < 2
- true

Comment (edplato) [reply web/js/natural-events/track.js]: it seems to work fine for me with the tropical storm in the mock data, but Mike's 
`Math.abs(activeCoord[0] - nextCoord[0]) > 180` may be easier to debug

Comment (Benjaki2) [reply web/js/natural-events/track.js]: Good call. Thanks y'all.

**Comment by edplato** [comment]
Looks like polar (at least antarctic) event tracks are gone. If you change `extent` back to `[-180, -90, 180, 90]` in `cluster.js:108`, the antarctic tracks are back, but then the mock event doesn't cross the dateline.

**Comment by Benjaki2** [comment]
Nice Find @edplato ! That should be working now.

**Pull request merged**